src/dodal/devices/i04/transfocator.py

```diff
@@ -0,0 +1,58 @@
+from time import sleep
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO, Kind
+from ophyd.status import DeviceStatus
+
+from dodal.log import LOGGER
+
+
+class Transfocator(Device):
+    beamsize_set: EpicsSignal = Cpt(EpicsSignal, "VERT_REQ", kind=Kind.hinted)
+    predict_vertical: EpicsSignal = Cpt(EpicsSignal, "LENS_PRED")
+
+    number_filters_sp: EpicsSignal = Cpt(EpicsSignal, "NUM_FILTERS")
+
+    start: EpicsSignal = Cpt(EpicsSignal, "START.PROC")
+    start_rbv: EpicsSignalRO = Cpt(EpicsSignalRO, "START_RBV")
+
+    vertical_lens_rbv: EpicsSignalRO = Cpt(EpicsSignalRO, "VER", kind=Kind.hinted)
+
+    TIMEOUT = 120
+    _POLLING_WAIT = 0.01
+
+    def polling_wait_on_stat_rbv(self, for_value):
```

Comment:
```suggestion
    def polling_wait_on_start_rbv(self, for_value):
```

Unless it's meant to be `stat_rbv`, in which case the field name (and PV name) are likely wrong